### PR TITLE
Move oc/lib/graph to staging

### PR DESCRIPTION
**Pull request by soltysh**

### CODE REVIEW

**Comment by damemi** [comment]
looks good but your govet is failing and `pkg/oc/cli/admin/prune/imageprune/prune_test.go` is still trying to import `github.com/openshift/origin/pkg/oc/cli/admin/prune/imageprune/testutil`

**Comment by soltysh** [comment]
Fix tests, self-tagging.

File: staging/src/github.com/openshift/oc/pkg/helpers/image/test/util.go [hunk 1]
```diff
@@ -33,6 +32,10 @@ var (
 	Config2 = "sha256:8ddc19f16526912237dd8af81971d5e4dd0587907234be2b83e249518d5b673f"
 )
 
+const (
+	managedByOpenShiftAnnotation = "openshift.io/image.managed"
```
Comment (deads2k) [inline staging/src/github.com/openshift/oc/pkg/helpers/image/test/util.go]: move to openshift/api and replace here.  You can have it as a followup don't forget.

**Comment by soltysh** [comment]
Fixed last unit, re-tagging.

File: staging/src/github.com/openshift/oc/pkg/helpers/image/test/util.go (hunk 1 above)
Comment (soltysh) [reply staging/src/github.com/openshift/oc/pkg/helpers/image/test/util.go]: @deads2k done in https://github.com/openshift/api/pull/338

**Comment by soltysh** [comment]
/hold
I need to check the tests results, if they are not related to this PR

**Comment by soltysh** [comment]
/hold cancel
The failures are related to cert issue from earlier today.

**Pull request merged**